website/docs/_posts/2018-04-09-configuration-templates.md

```diff
@@ -0,0 +1,51 @@
+---
+date: 2018-04-09
+title: Config Template Editing
+categories:
+  - deep-dives
+description: How to edit Renovate's config templates
+type: Document
+order: 30
+---
+
+Renovate has always been intended to be as configurable as possible and therefore supported handlebars templates for branch name, commit message, PR title and PR body from almost the very start. However, with great power also comes great potential to shoot yourself in the foot.
+
+Read on for some recommendations of what to do if you want to change an of the existing templates.
```

Comment:
`an` => `any`

Reply:
Though I understand, after reading both paragraphs, that the point of this article is to explain how to customize the branch name, commit message, etc., these paragraphs don't just come out and say `Here's how you customize the commit message, branch name, etc. for Renovate...`